tests/__main__.py

```diff
@@ -7,9 +7,10 @@
 os.chdir(REPO_ROOT)
 
 os.putenv(
-    "PATH", "{}:{}".format(os.path.join(REPO_ROOT, "bin"), os.getenv("PATH"))
+    "PATH", ":".join([os.path.join(REPO_ROOT, "bin"), os.getenv("PATH")])
 )
 os.putenv("DVC_HOME", REPO_ROOT)
+os.putenv("TQDM_DISABLE", "False")
```

Comment:
Double negation. Also flag name is not what it is about, this is about forcing it to ignore isatty test, not about enabling/disabling it.

Reply:
Not currently. It really *is* an override of all `Tqdm.disable` instances. Double negation would be preferable to inconsistency (e.g. `disable=not os.environ.get("TQDM_ENABLE")` would be inconsistent parameter naming and delayed double negation).

If instead you want an env var to just override `isatty`, then perhaps we could instead do something like this:

```python
getattr(file, "isatty", lambda: os.environ.get("ISATTY", False))()
```